examples/motorkit_dc_test.py

```diff
@@ -27,13 +27,13 @@
     time.sleep(1)
 
     print("Speed up...")
-    for i in range(-100, 0):
+    for i in range(0, -100, -1):
```

Comment:
Same potential issue here:

Current:
```
>>> for i in range(-10, 0):
...     print(i)
...
-10
-9
-8
-7
-6
-5
-4
-3
-2
-1
```
Proposed change:
```
>>> for i in range(0, -10, -1):
...     print(i)
...
0
-1
-2
-3
-4
-5
-6
-7
-8
-9
>>>
```

Reply:
In this case, full reverse is a throttle value of -1.0, not 0 as in the original code. However, the revised version should be corrected to
`for i in range(0, -11, -1):`
so that the calculated throttle value ranges from 0 (stopped) to -1.0 (full reverse).
![image](https://user-images.githubusercontent.com/29906257/52905830-bfbb1180-31f4-11e9-901f-73a79e72218f.png)